docs/mkdocs/docs/api/macros/nlohmann_json_namespace_version_namespace.md

```diff
@@ -0,0 +1,46 @@
+# NLOHMANN_JSON_NAMESPACE
+
+```cpp
+#define NLOHMANN_JSON_NAMESPACE_VERSION_NAMESPACE /* value */
+```
+
+This macro evaluates to the name of the version namespace. See [`NLOHMANN_JSON_NAMESPACE`](nlohmann_json_namespace.md).
+If defined to an empty value, the version namespace will be omitted.
```

Comment:
Can we do just `NLOHMANN_JSON_NAMESPACE_EXCLUDE_VERSION` and if it's defined to 1, we leave out the version?

Reply:
That might be the easier option. I just wanted to avoid yet another page of documentation. Although, I think I can document `NLOHMANN_JSON_NAMESPACE_EXCLUDE_VERSION` on the same page.

I'll think about it some more.

Reply:
Thanks. I needed someone to make me stop being stubborn and think for a second. :-)

Reply:
I was assuming that this would replace this one.

Reply:
Yes, exactly.